src/ticket/Ticket.java

```diff
@@ -0,0 +1,28 @@
+package ticket;
+
+import datetimeutils.DateTimeUtils;
+
+public class Ticket {
+
+    public long fineID;
+    private final GovAuthorities authorityName;
+    public DateTimeUtils issueDate;
+    public Boolean isPaid;
+    public double fine;
+    private static final double UMA = 207.44;
+
+    public Ticket(long fineID, DateTimeUtils issueDate, GovAuthorities authorityName, Boolean isPaid) {
+        this.fineID = fineID;
```

Comment:
Seria mejor que se inicialice con un id por automatico. Algún numero random, en vez de tu tener que pasarle explicitamente el id que quieres.

```suggestion
        this.fineID = generateRandomID();
```